lib/workers/evm_server.rb

```diff
@@ -49,6 +49,7 @@ def monitor_servers
     loop do
       refresh_servers_to_monitor
       as_each_server { monitor }
+      messaging_health_check
       sleep ::Settings.server.monitor_poll.to_i_with_method
     end
```

Comment:
Not sure if this will cause a lot of overhead, due to the frequency of health checks

Reply:
Yeah checking this every 5 seconds is probably excessive.

We could check this in the `server_monitor` block in `MiqServer`, @kbrock wdyt?

Reply:
Yea, that is every 60 seconds, which is more reasonable.

You thinking in `monitor_servers_as_non_master`? https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_server/server_monitor.rb#L96